README.md

```diff
@@ -78,6 +78,25 @@ To launch the interactive CLI, run the executable without any arguments.
 
 Run `ficsit help` to see a list of available commands.
 
+## Game Installations
+
+The app needs to know where the game is installed. On Steam, this will be
+
+    C:\Program Files (x86)\Steam\steamapps\common\Satisfactory
+
+When running on Windows, it should auto-detect this path and you have nothing to worry about.   
+
+But this won't be the case When running on MacOS with Satisfactory installed in Crossover.
+In this case you have to add the installation manually.
+The path to a crossover-installed game, assuming a bottle named "Steam" will be
+
+    ~/Library/Application\ Support/CrossOver/Bottles/Steam/drive_c/Program\ Files\ \(x86\)/Steam/steamapps/common/Satisfactory
+
+The command to add an installation is then:
```

Comment:
```suggestion
The app needs to know where the game is installed to function. 

If you're not sure where that is, see the [Satisfactory Modding FAQ section about locating your game install](https://docs.ficsit.app/satisfactory-modding/latest/faq.html#Files_GameInstall) to learn approaches for finding out.

To add an installation, either use the interactive interface, or use the command:
```

Reply:
That FAQ **DOES NOT HELP** on MacOS.  It still only shows the path in Windows-format as it appears *inside the bottle*, it still forces people to manually translate that path to the path on the MacOS filesystem containing that bottle, and it doesn't give a quick-and-easy example of the default path that 99% of users will see.

If you want to reject this PR on that basis then so be it, but I'm not prepared to accept a proposed change that **WILL** make this tool harder to use on MacOS.